src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -76,9 +76,16 @@
     <Compile Include="$(CommonPath)\System\IO\PathInternal.CaseSensitivity.cs">
       <Link>Common\System\IO\PathInternal.CaseSensitivity.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Threading\Tasks\TaskToApm.cs">
+      <Link>Common\System\Threading\Tasks\TaskToApm.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <!-- .NET Standard 1.7 -->
+  <ItemGroup Condition="'$(PackageTargetFramework)' == 'netstandard1.7' and '$(TargetGroup)' != 'net46' and '$(TargetGroup)' != 'net463'" >
```

Comment:
We are trying to be consistent on these conditional file includes and use TargetGroup or OSGroup/TargetsOSXYZ for these. Which is why I suggest the change as opposed to using PackageTargetFramework. It will make it much easier to identify all the things that are included for the default build which is TargetGroup empty.